core/domain/rte_component_registry_test.py

```diff
@@ -43,7 +43,7 @@
 RTE_THUMBNAIL_HEIGHT_PX: Final = 16
 RTE_THUMBNAIL_WIDTH_PX: Final = 16
 
-# Here, we used object because every type is inherited from object class.
+# Here we use object because every type is inherited from object class.
```

Comment:
we cannot define Union[bool, list, str], because here we have a generic type list without type arguments and as explained above in the comment -- https://github.com/oppia/oppia/pull/15905#discussion_r966333874. Currently, we cannot use generics without type arguments.